tgui/packages/tgui/interfaces/Processor.js

```diff
@@ -0,0 +1,109 @@
+import { useBackend } from '../backend';
+import { Box, Button, Flex, Knob, LabeledList, Section } from '../components';
+import { Window } from '../layouts';
+
+export const Processor = (props, context) => {
+  const { act, data } = useBackend(context);
+  // Extract `health` and `color` variables from the `data` object.
+  const {
+    materials_data = [],
+    alloy_data = [],
+    currently_alloying,
+    running,
+    sheet_rate,
+  } = data;
+  return data.machine ? (
+
+    <Window resizable>
+      <Window.Content scrollable>
+        <Flex
+          frex-wrap="wrap"
+          >
+        <Flex.Item>
```

Comment:
indentation. eslint will complain, vscode should show the error too (its fixable)